standup_notes/__main__.py

```diff
@@ -83,14 +90,56 @@ def copy_note(day: date):
         print(note + ' doesn\'t exist yet.')
 
 
+def copy_prev(day: date):
+    previous_days_note = get_note_name(last_weekday(day))
+    note = get_note_name(day)
+    date_of_note = "Date: " + day.strftime("%m/%d/%Y") + " \n"
+    beginning_format = "__What I did yesterday__:\n"
+    end_format = "__What I'm doing today__:\n \n__Blockers__:\n-none\n"
+    flag = False
+    lines_to_append = []
+
+    if os.path.exists(previous_days_note):
+        result = verify_input("Found yesterdays notes, would you like to insert applicable information into your "
+                              "notes y/n: ")
+        if result:
+            with open(previous_days_note) as f:
+                for line in f:
+                    if flag:
+                        if '__Blockers__:' not in line:
+                            lines_to_append.append(line)
+                    if '__What I\'m doing today__:' in line:
+                        flag = True
+                    if '__Blockers__:' in line:
+                        flag = False
+            if os.path.exists(note):
```

Comment:
I'm a little lost, what's the behaviour if the day you're trying to edit already exists and contains notes?

Reply:
The behavior is that it will append the previous day's notes into "What I did yesterday" of that file if it exists. It will not change anything else in the file.

Reply:
Could you print that our so it's clear to the user?